e2e/core/ui/CommonActions.js

```diff
@@ -11,6 +12,7 @@ class CommonActions {
      * Method to wait for an element to be visible to get the text.
      * @param elementCSS WebElement locator.
      */
+    //TODO remove wait from methods names
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)